<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlightForm</class>
 <widget class="QWidget" name="FlightForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Flight&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Flight Information</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Flight Id</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="flightIDLineEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Producer's Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="producerNameLineEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Description</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="descriptionTextEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Date of Flight Execution</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="flightDateEdit">
               <property name="specialValueText">
                <string/>
               </property>
               <property name="minimumDate">
                <date>
                 <year>1913</year>
                 <month>1</month>
                 <day>1</day>
                </date>
               </property>
               <property name="displayFormat">
                <string>dd/MM/yyyy</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Flight Parameters</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Nominal Flight Altitude Above Sea Level (m)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="flightHeightDoubleSpinBox">
               <property name="toolTip">
                <string/>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="specialValueText">
                <string/>
               </property>
               <property name="accelerated">
                <bool>true</bool>
               </property>
               <property name="maximum">
                <double>100000.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Nominal Scale</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="nominalScaleSpinBox">
               <property name="toolTip">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Example: 1:10.000 equals &quot;One to Ten Thousand Scale&quot;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="whatsThis">
                <string/>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="specialValueText">
                <string/>
               </property>
               <property name="accelerated">
                <bool>true</bool>
               </property>
               <property name="prefix">
                <string>1:</string>
               </property>
               <property name="minimum">
                <number>99</number>
               </property>
               <property name="maximum">
                <number>10000000</number>
               </property>
               <property name="singleStep">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Longitudinal Overlap (%)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="br::uerj::eng::efoto::EfotoDoubleSpinBox" name="horizontalOverlapDoubleSpinBox">
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="specialValueText">
                <string/>
               </property>
               <property name="accelerated">
                <bool>true</bool>
               </property>
               <property name="minimum">
                <double>4.000000000000000</double>
               </property>
               <property name="maximum">
                <double>95.000000000000000</double>
               </property>
               <property name="value">
                <double>5.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Transversal Overlap (%)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="br::uerj::eng::efoto::EfotoDoubleSpinBox" name="verticalOverlapDoubleSpinBox">
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="specialValueText">
                <string/>
               </property>
               <property name="accelerated">
                <bool>true</bool>
               </property>
               <property name="minimum">
                <double>4.000000000000000</double>
               </property>
               <property name="maximum">
                <double>95.000000000000000</double>
               </property>
               <property name="value">
                <double>5.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>br::uerj::eng::efoto::EfotoDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>EfotoDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
